tests/php/Http/Controllers/Api/Volumes/MetadataControllerTest.php

```diff
@@ -60,15 +60,15 @@ public function testStoreImageMetadata()
         $png = $png->fresh();
         $jpg = $jpg->fresh();
 
-        $this->assertEquals('2016-12-19 12:27:00', $jpg->taken_at);
-        $this->assertEquals(52.220, $jpg->lng);
-        $this->assertEquals(28.123, $jpg->lat);
-        $this->assertEquals(-1500, $jpg->metadata['gps_altitude']);
-        $this->assertEquals(2.6, $jpg->metadata['area']);
+        $this->assertSame('2016-12-19 12:27:00', $jpg->taken_at->toDateTimeString());
+        $this->assertSame(52.220, $jpg->lng);
+        $this->assertSame(28.123, $jpg->lat);
+        $this->assertSame(-1500, intval($jpg->metadata['gps_altitude']));
+        $this->assertSame(2.6, floatval($jpg->metadata['area']));
         // Import should update but not destroy existing metadata.
-        $this->assertEquals(10, $jpg->metadata['distance_to_ground']);
-        $this->assertEquals(4000, $jpg->metadata['water_depth']);
-        $this->assertEquals(180, $jpg->metadata['yaw']);
+        $this->assertSame(10, intval($jpg->metadata['distance_to_ground']));
+        $this->assertSame(4000, $jpg->metadata['water_depth']);
+        $this->assertSame(180, intval($jpg->metadata['yaw']));
```

Comment:
Please update the metadata code so these values are actually stored as int/float.